homeassistant/components/romy/config_flow.py

```diff
@@ -0,0 +1,106 @@
+"""Config flow for ROMY integration."""
+from __future__ import annotations
+
+from typing import Any, Optional
+
+import romy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import zeroconf
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, LOGGER
+
+
+def _schema_with_defaults(
+    default_values: Optional[dict[str, Any]] = None,
+    requires_password: bool = False,
+) -> vol.Schema:
+    if default_values is None:
+        default_values = {}
+    schema = {
+        vol.Required(CONF_HOST, default=default_values.get(CONF_HOST, "")): cv.string,
+        vol.Optional(CONF_NAME, default=default_values.get(CONF_NAME, "")): cv.string,
+    }
+
+    if requires_password:
+        schema[vol.Required(CONF_PASSWORD)] = vol.All(str, vol.Length(8))
+
+    return vol.Schema(schema)
+
+
+class RomyConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle config flow for ROMY."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self) -> None:
+        """Handle a config flow for ROMY."""
+        self.discovery_schema = None
+
+    async def async_step_user(
```

Comment:
Please set also a unique id when the user manually start the config flow

Reply:
setting it now in user control flow